index.js

```diff
@@ -67,6 +67,42 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener("click", function() {
+    toggleColor(header)
+})
```

Comment:
great work! We have the header variable already defined earlier in the file.
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered. It is ok that you used header as the argument to the toggleColor function on line 71. I would also get comfortable using the event target as well.


Great work incorporating the `toggleColor` function. We just need to pass in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```

Great work incorporating the `toggleColor` function. We just need to pass in the event target as an argument to the function.

```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})

```

With ES6 notation

```
header.addEventListener("click", e => toggleColor(e.target))
```